sites/example-project/src/components/viz/ErrorChart.svelte

```diff
@@ -17,9 +17,9 @@
 			class:w-[7.8em]={chartType.includes('Value')}
 		>
 			{error}
+			{#if dev && error === MissingCredentialsError}
+				<br><a class="credentials-link" href="/settings"> Add credentials &rarr;</a>
```

Comment:
Should we add a non breaking space between the word "credentials" and the right arrow?
In your screenshot there appears to be a linebreak.

(Or else we could remove the right arrow completely)

Reply:
Good point, looks nicer with no breaks:
<img width="180" alt="image" src="https://github.com/evidence-dev/evidence/assets/61282104/d331517c-28c8-4cc9-937b-099ecc1f83a8">